### PR TITLE
Remove Auth header in start/stopRequest for rex

### PR DESCRIPTION
**Pull request by thescouser89**
Thanks to @michalovjan for pointing this out!

### Checklist:

* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by thescouser89** [comment]
retest this please

**Comment by michalovjan** [comment]
@thescouser89 the MyHeaderPropagator is for request from Orch -> Rex so there must be some Authorization header. The place for Start/StopRequest Authorization headers is in RexFacade getCreateNewTaskDTO, which is the place where the headers should be removed.

File: remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/rexclient/MyHeaderPropagator.java [hunk 1]
```diff
@@ -41,12 +41,6 @@ public MultivaluedMap<String, String> update(
             MultivaluedMap<String, String> clientOutgoingHeaders) {
         MultivaluedMap<String, String> outgoingHeaders = new MultivaluedHashMap<>();
 
-        // propagate User Token if present in original request
-        if (incomingHeaders.containsKey(HttpHeaders.AUTHORIZATION_STRING)) {
```
Comment (michalovjan) [inline remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/rexclient/MyHeaderPropagator.java:45]: In this place, there should be Authorization header from Orch's SA

**Review by michalovjan — CHANGES_REQUESTED** [review]
@thescouser89 the MyHeaderPropagator is for request from Orch -> Rex so there must be some Authorization header. The place for Start/StopRequest Authorization headers is in RexFacade getCreateNewTaskDTO, which is the place where the headers should be removed.

File: remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/rexclient/MyHeaderPropagator.java (hunk 1 above)
Comment (thescouser89) [reply remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/rexclient/MyHeaderPropagator.java:45]: @michalovjan Thanks for the pointer! Done :)

**Comment by thescouser89** [comment]
retest this please

**Comment by michalovjan** [comment]
retest this please

**Review by michalovjan — APPROVED** [review]

**Pull request merged**